values.yaml

```diff
@@ -51,15 +51,20 @@ sts:
 properties:
   # use externalSecure for when inbound SSL is provided by nginx-ingress or other external mechanism
   externalSecure: false
-  isNode: true
-  httpPort: 8080
-  httpsPort: null
-  clusterPort: 6007
-  clusterSecure: false
+  isNode: false
+  isSecure: false # switch to true if the cluster is secured ( if you're using Ldap for example )
+  webProxyHost:
+  webHttpsHost:
+  release: nifi # if you're using the secured cluster, provide the release-name here before deploying.
```

Comment:
use `hostname -f` in your scripts, so the user do not need to use these variables.

Reply:
And if you are needing to use them in a template somewhere helm has built in objects that provide some of this I think: https://helm.sh/docs/chart_template_guide/builtin_objects/